WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -701,8 +701,18 @@ private extension DashboardViewController {
 
     func onPullToRefresh() async {
         ServiceLocator.analytics.track(.dashboardPulledToRefresh)
-        await viewModel.syncAnnouncements(for: siteID)
-        await reloadDashboardUIStatsVersion(forced: true)
+        await withTaskGroup(of: Void.self) { group in
```

Comment:
nice use of the task group! 💯 the old way was still doing one thing after another.